src/main/java/org/fundacionjala/coding/nestor/NewReleaseMovie.java

```diff
@@ -0,0 +1,17 @@
+package org.fundacionjala.coding.nestor;
+
+/**
+ *
+ */
+public class NewReleaseMovie extends Movie {
+
+    public NewReleaseMovie(String title) {
+        super(title, MovieType.NEW_RELEASE);
+    }
+
+    @Override
+    public double getPrice(int daysRented) {
+        return daysRented * DAYS;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)